samples_project/Assets/SampleViewer/Samples/Geocoding/readme.md

```diff
@@ -1,19 +1,29 @@
-# Geocoding
+# Find places or search for an address
 
```

Comment:
I think we can just use "Geocoding" here since it is a verb, and geocoding is a common terminology, it's kind of strange to not use terminology as a GIS company